website/docs/docs/cloud/git/connect-gitlab.md

```diff
@@ -71,6 +71,8 @@ The application form in GitLab should look as follows when completed:
 
 Click **Save application** in GitLab, and GitLab will then generate an **Application ID** and **Secret**. These values will be available even if you close the app screen, so this is not the only chance you have to save them.
 
+If you're a Business Critical customer using IP restrictions ensure you've added the appropriate Gitlab CIDRs to your IP restriction rules, or else the Gitlab connection will fail.
```

Comment:
@saimaddali I've added these lines to the bottom of the gitlab and ADO integration sections - would you rather these be in a more visible place?

Reply:
Let's also add it to the ip restriction docs (`website/docs/docs/cloud/secure/ip-restrictions.md`)